docs/src/main/asciidoc/spanner.adoc

```diff
@@ -404,6 +404,22 @@ A table may have multiple child tables.
 On updating or inserting an object to Cloud Spanner, all of its referenced children objects are also updated or inserted in the same request, respectively.
 On read, all of the interleaved child rows are also all read.
 
+===== Lazy Fetch
+
+`@Interleaved` properties are retrieved eagerly by default, but can be fetched lazily for performance in both read and write:
+
+[source,java]
+----
+@Interleaved(lazy = true)
+List<Album> albums;
+----
+
+Lazy interleaved properties are retrieved upon the first interaction with the property.
+If a lazy property is never retrieved, then it is also skipped when saving the parent entity.
+
+If used inside a transaction, subsequent operations on lazy properties use the same transaction context as that of the original parent entity.
```

Comment:
I'm not a fan of the "lazy property" wording. I know it's more succinct, but it just doesn't sound as good as lazy fetch.

Reply:
@meltsufin fixed all such instances.